### PR TITLE
Add documentation about incorrect VPC diffing

### PR DESCRIPTION
**Pull request by yissachar**
See https://github.com/kubernetes/kops/issues/476

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kops/1022)

### CODE REVIEW

**Comment by chrislovecnm** [comment]
@yissachar think you used the incorrect email address, linuxfoundation cla not happy

File: docs/run_in_existing_vpc.md [hunk 1]
```diff
@@ -45,6 +45,19 @@ kops update cluster ${CLUSTER_NAME}
 Review the changes to make sure they are OK -  the Kubernetes settings might not be ones you want on a shared VPC (in which case,
 open an issue!)
 
+There is currently a [bug[(https://github.com/kubernetes/kops/issues/476) where kops will tell you that it will modify the VPC and InternetGateway names:
```
Comment (chrislovecnm) [inline docs/run_in_existing_vpc.md]: Does PR fix this? https://github.com/kubernetes/kops/pull/997

Comment (chrislovecnm) [inline docs/run_in_existing_vpc.md]: Can we make a note in the PR to remove this copy when the bug is closed?

**Comment by yissachar** [comment]
I signed it!

File: docs/run_in_existing_vpc.md (hunk 1 above)
Comment (yissachar) [inline docs/run_in_existing_vpc.md]: I don't believe that fixes this. I have a note that says it is related to https://github.com/kubernetes/kops/issues/192, which I think @justinsb mentioned.

Not sure how I feel about adding TODO items in user facing docs. How about we add a note in the bug instead?

Comment (chrislovecnm) [inline docs/run_in_existing_vpc.md]: Yah not sure myself ;) Let me ponder

Comment (kris-nova) [inline docs/run_in_existing_vpc.md]: I added a note in the open issue to make sure we update the docs

**Comment by kris-nova** [comment]
LGTM - as long as we remember to remove it when we work on issue https://github.com/kubernetes/kops/issues/192

**Pull request merged**